src/frontend/components/SøknadsSteg/OmBarnaDine/OmBarnaDine.tsx

```diff
@@ -201,16 +203,22 @@ const OmBarnaDine: React.FC = () => {
                     }
                 />
                 <HvilkeBarnCheckboxGruppe
-                    legendSpråkId={
-                        omBarnaDineSpørsmålSpråkId[OmBarnaDineSpørsmålId.hvemBarnehageplass]
-                    }
-                    skjemafelt={skjema.felter.hvemBarnehageplass}
+                    legend={<TekstBlock block={hvemBarnehageplass.sporsmal} />}
+                    skjemafelt={skjema.felter.hvemHarBarnehageplass}
                     søknadsdatafelt={barnDataKeySpørsmål.harBarnehageplass}
                     nullstillValgteBarn={
                         skjema.felter.harEllerTildeltBarnehageplass.verdi === ESvar.NEI
                     }
                     visFeilmelding={skjema.visFeilmeldinger}
-                />
+                >
+                    {hvemBarnehageplass.vedleggsnotis ? (
+                        <VedleggNotis>
+                            <Normaltekst>
+                                {localeString(hvemBarnehageplass.vedleggsnotis)}
+                            </Normaltekst>
```

Comment:
Måtte wrappe dette i normaltekst for at det skulle aligne med ikonet.

Ren tekst:
<img width="243" alt="image" src="https://user-images.githubusercontent.com/17828446/191735095-e7b90541-2139-4e80-bf25-ac29b362d99e.png">

Med normaltekst:
<img width="241" alt="image" src="https://user-images.githubusercontent.com/17828446/191735142-3e2e77ad-049a-4e6c-bbbb-593a681970cf.png">

Reply:
Kanskje dette kan håndteres inne i VedleggNotis feks med flex eller noe sånn?